lib/accumulate_distribute/meta/validate_params.js

```diff
@@ -77,6 +77,10 @@ const validateParams = (args = {}, pairConfig = {}) => {
     return validationErrObj('limitPrice', 'Limit price required for LIMIT order type')
   }
 
+  if (orderType === 'LIMIT' && !_isBoolean(postonly)) {
```

Comment:
should postonly be always true in case of limit order?

Reply:
nope.. it can be false as well .. its up to the users on how they wanna submit the order